package.json

```diff
@@ -9,7 +9,7 @@
     "dev": "node launch-dev.js",
     "watch": "watchify src/loop54.js -o lib/loop54-js-connector.js -t [ babelify --presets [ @babel/preset-env ] ]",
     "server": "node server.js",
-    "build-connector": "browserify src/loop54.js -o lib/loop54-js-connector.js -t [ babelify --presets [ @babel/preset-env ] ]",
+    "build-connector": "browserify src/loop54.js -o lib/loop54-js-connector.js -t [ babelify --global --ignore '[/node_modules/(?!axios/)]' --presets [ @babel/preset-env ] ]",
```

Comment:
What happens if we omit ignoring axios in babelify?

Reply:
It's actually including axios, by default everything in node_modules is ignored by babel. We want to ignore everything EXCEPT axios. Axios v1 uses ES modules so we need to transpile it using babel.